contrib/registry/eureka/client.go

```
@@ -334,12 +338,7 @@ func (e *Client) request(ctx context.Context, method string, params []string, in
 			return false, err
 		}
 	}
-
-	if resp.StatusCode >= http.StatusBadRequest {
-		return false, fmt.Errorf("response Error %d", resp.StatusCode)
-	}
-
-	return false, nil
+	return false, fmt.Errorf("response error %d", resp.StatusCode)
```

all but 200 or 204 status should return an error?